src/content/docs/accounts/original-accounts-billing/original-data-retention/event-data-retention-original-pricing-plan.mdx

```diff
@@ -187,7 +177,7 @@ With an Insights Pro subscription, you can use [flexible retention to customize
 
 ## Flexible data retention [#flex-retention]
 
-With an [Insights Pro subscription](#insights-pro), you get access to **flexible retention**, which lets you define how some types of [event data](/docs/insights/use-insights-ui/manage-account-data/insights-data-retention) are retained. This lets you keep only the event data you need, for as long as you need it. You can manage your flexible retention [through the UI](#ui-overview) or [through our GraphQL API](#api-examples).
+For organizations on our original pricing model, a subscription to [Insights Pro](#insights-pro) gets you access to **flexible retention**, which lets you define how some types of [event data](/docs/insights/use-insights-ui/manage-account-data/insights-data-retention) are retained. This lets you keep only the event data you need, for as long as you need it. You can manage your flexible retention [through the UI](#ui-overview) or [through our GraphQL API](#api-examples).
```

Comment:
Super minor - We're using "this lets" twice. You can change the first one to "allows", as in "to **flexible retention**, which allows you define how".